README.md

```diff
@@ -335,7 +353,12 @@ module.exports = {
 
 #### `filter`
 
-Type: `Function`
+Type:
+
+```ts
+type filter = (filepath: string) => any;
```

Comment:
Should return boolean

Reply:
- https://github.com/webpack-contrib/copy-webpack-plugin/blob/9b5a314b71e9ab16ad3ada0816620aa75d18ca47/types/index.d.ts#L252

I took it from here?

Reply:
Let's fix our types too :smile:

Reply:
Agreed